configure.ac

```diff
@@ -6,7 +6,7 @@ AM_INIT_AUTOMAKE([1.11 subdir-objects])
 AM_SILENT_RULES([yes])
 AM_MAINTAINER_MODE([disable])
 
-VERSION_INFO="4:0:0"
+VERSION_INFO="5:0:0"
```

Comment:
It's an output-only type for us, so even if a user allocates a FFMS_FrameInfo without the new field there aren't actually any functions that they can pass it to.